sixtyfps_compiler/llr/item_tree.rs

```diff
@@ -167,6 +167,44 @@ impl TreeNode {
             }
         }
     }
+
+    pub fn sub_component<'a>(&self, root_sub_component: &'a SubComponent) -> &'a SubComponent {
+        let mut current_component = root_sub_component;
+        for sub_component_index in &self.sub_component_path {
+            current_component = &(current_component
+                .sub_components
+                .get(*sub_component_index)
+                .expect("Illegal sub component index in TreeNode")
+                .ty);
+        }
+        current_component
+    }
+
+    pub fn item<'a>(&self, root_sub_component: &'a SubComponent) -> Option<&'a Item> {
+        if self.repeated {
+            None
+        } else {
+            let sub_component = self.sub_component(root_sub_component);
+            Some(sub_component.items.get(self.item_index).expect("Illegal item index in TreeNode"))
+        }
+    }
+
+    pub fn repeated_element<'a>(
```

Comment:
AFAICS these three functions are not called - I think that we can remove them again.